src/pages/change-password/index.tsx

```diff
@@ -76,41 +95,32 @@ const Index = () => {
               In order to access the application you need to change your password and set your
               username.
```

Comment:
Perhaps change this to something more welcoming like:

`Welcome to the application of The Rhapsodies! Please give us your name and change your password.`